public/src/js/widgets/fetch-latest-packages.js

```diff
@@ -19,7 +19,16 @@ export default class extends Extension {
                 isHidden: this.baseModel.priority === 'training'
             }
         })
-        .then(response => this.baseModel.latestPackages(response.results))
+        .then(result => {
+            return this.baseModel.latestPackages(result.response.results.map(result => {
+                var newResult = {
```

Comment:
I trust you that the mapping is correct :smile: 
there's no need to create a new var, you can simply `return`